README.rst

```diff
@@ -76,8 +76,49 @@ Run tests
 To check your code, run `make checks`. It will run style checks and tests. It's also possible to
 run sub-target independently:
 
-- `make checks-style` to run only style-related checks (linting).
-- `make tests` to run tests. Python 3.7 is used by default. But you can run tests with another version by
-  running: `PYTHON_TEST_VERSION=python3.x make tests`. You can also set this variable in the Makefile.
+- ``make checks-style`` to run only style-related checks (linting).
+- ``make tests`` to run tests. Python 3.7 is used by default. But you can run tests with another version by
+  running: ``PYTHON_TEST_VERSION=python3.x make tests``. You can also set this variable in the Makefile.
 
-After running tests, the coverage is available in the folder `coverage_report`.
+After running tests, the coverage is available in the folder ``coverage_report``.
+
+Use Oereblex data model
+-----------------------
+
+To test the application with the Oereblex data model, adapt the configuration files ``pyramid_oereb_standard.yml.mako`` and ``docker/config.yml.tmpl``.
+See this example with the PLR forest_perimeters:
+
+.. code-block:: yaml
+
+  source:
+    # Standard data model
+    #class: pyramid_oereb.standard.sources.plr.DatabaseSource
+    # Oereblex data model
+    class: pyramid_oereb.contrib.sources.plr_oereblex.DatabaseOEREBlexSource
+    params:
+      db_connection: *main_db_connection
+      # Standard data model
+      #models: pyramid_oereb.standard.models.forest_perimeters
+      # Oereblex data model
+      models: pyramid_oereb.contrib.models.oereblex.forest_perimeters
+
+The sample data is in ``sample_data/oereblex``. 
+
+Run the application:
+
+.. code-block:: bash
+
+  USE_OEREBLEX=TRUE make serve
+
+
+JSON reduced extract is accessible at: http://localhost:6544/oereb/extract/reduced/json/CH113928077734
+
+It is possible to run this instance in parallel to the instance which uses the standard database. For this, one should create a second clone of the project.
+
+To run the tests:
+
+.. code-block:: bash
+
+  make tests
```

Comment:
When using the configuration described here, the Oereblex model is used for the theme ForestPerimeters in the test database. Some modifications were required for the tests to run with this configuration (otherwise you need to change again one of the configuration file between make serve and make tests which I thought was not ideal). But I don't think there are specific tests done with this data (I did not add any), so nothing changes really for the tester. I am not sure what we should add here?